docs/summary.md

```diff
@@ -12,15 +12,13 @@
 ## Deployments
 
 * [Deployments](deployments/deployments.md)
-* [Predictor APIs](deployments/predictor.md)
+* [Python APIs](deployments/python.md)
 * [TensorFlow APIs](deployments/tensorflow.md)
 * [ONNX APIs](deployments/onnx.md)
-* [Request handlers](deployments/request-handlers.md)
 * [Autoscaling](deployments/autoscaling.md)
 * [Prediction monitoring](deployments/prediction-monitoring.md)
 * [Compute](deployments/compute.md)
 * [API statuses](deployments/statuses.md)
-* [Python client](deployments/python-client.md)
```

Comment:
It might be better to remove this in a separate commit so we can easily see the diff if/when we want to add it back